avaframe/com1DFA/com1DFA.py

```diff
@@ -1607,9 +1607,7 @@ def initializeResistance(cfg, dem, simTypeActual, resLine, reportAreaInfo, thres
     reportAreaInfo: dict
         simulation area information dictionary completed with entrainment area info
     """
-    d = cfg.getfloat("dRes")
-    cw = cfg.getfloat("cw")
-    sres = cfg.getfloat("sres")
+    cres = cfg.getfloat("cres")
```

Comment:
I suggest: while we're at it, let's switch to our convention and use cRes instead of cres everywhere